src/Microsoft.CodeAnalysis.Analyzers/Test/MetaAnalyzers/StartActionWithOnlyEndActionRuleTests.cs

```diff
@@ -340,5 +398,12 @@ private static DiagnosticResult GetExpectedDiagnostic(string language, int line,
                 }
             };
         }
+
+        private enum StartActionKind
```

Comment:
The enum kind basically corresponds to the actions for which the given rule is fired - they just happen to currently have the same values across both rules but this might differ in future. I'd rather keep these specific to test files for now.